content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,302 @@
+related to *all* leads.
+
+Custom filters can now be added by clicking the :icon:`fa-caret-down` :guilabel:`(down caret)` icon,
+to the right of the search bar, to reveal a drop-down menu of search and filter options.
+
+Three columns are displayed: :guilabel:`Filters`, :guilabel:`Group By`, and :guilabel:`Favorites`.
+To begin, navigate to the bottom of the :guilabel:`Filters` column, and click :guilabel:`Add Custom
+Filter`. This opens an :guilabel:`Add Custom Filter` pop-up window, where a custom filter can be
+created.
+
+.. image:: lead_distribution_report/add-custom-filters.png
+   :align: center
+:alt: The Add Custom Filter pop-up window allows custom filtering rules to be created.
+
+.. _essential-filters:
+
+Essential filters
+-----------------
+
+The following filter conditions are used to create a basic lead distribution report. Together they
+gather all leads created within a certain timespan that have an associated contact method and have
+been assigned to a sales team.
+
+1. Lead creation date
+~~~~~~~~~~~~~~~~~~~~~
+
+Click the first field, under :guilabel:`Match any of the following rules:`, that has the value
+:guilabel:`Country` in it. A pop-up window appears. In that pop-up window, type `Created on` in the
+search bar, or scroll to search through the list to locate and select it.
+
+Then, in the second field of the :guilabel:`Add Custom Filter` pop-up window, select :guilabel:`>=`
+from the drop-down menu. This operator **only** includes values greater than (or equal to) the value
+in the third, rightmost field.
+
+The third field on the :guilabel:`Add Custom Filter` pop-up window should contain the earliest date
+leads are selected from.
+
+For example, setting `01/01/2024 00:00:00` only includes leads created from, and including, the
+first day of 2024.
+
+.. image:: lead_distribution_report/created-on.png
+  :align: center
+  :alt: Add a Created On rule for the start of the year onward.
+
+.. _team-location:
+
+2. Sales team location
+~~~~~~~~~~~~~~~~~~~~~~
+
+Click :guilabel:`New rule` to add a new set of rule fields to the :guilabel:`Add Custom Filter`
+pop-up window. Click the first field for the new rule, and set it to :guilabel:`Sales Team`. Then,
+click the second field of the new rule, and select :guilabel:`contains` from the drop-down menu.
+Selecting this operator filters for any records that contain the words in the third, rightmost
+field.
+
+In this third field, enter the name of the desired sales team(s). For example, setting `us direct sf
+northam` **only** includes sales teams with titles containing *US*, *direct*, *San Francisco*, and
+*North America*.
+
+.. image:: lead_distribution_report/sales-team-location.png
+  :align: center
+  :alt: Use Sales Team to filter the location the lead is associated with.
+
+By adding this new rule, a new line appears at the top of the pop-up menu showing :guilabel:`Match
+any of:` with a :icon:`fa-caret-down` :guilabel:`(down caret)` icon.
```

Comment:
I'd highlight this in an admonition to really make it stand out (it gets missed all of the time 😅 )

```suggestion
.. important::
   By adding more than one rule to the form, a new option emerges at the top of the pop-up window
   above all the filters, to specify whether :guilabel:`any` :icon:`fa-caret-down` :guilabel:`(down
   caret)` or :guilabel:`all` :icon:`fa-caret-down` :guilabel:`(down caret)` of the conditions
   should match. This distinction is important to set correctly, as it impacts the driving logic of
   how the filters return data.
```